### ISSUE TITLE
Failed to convert extracted value to float64" path="{ .results }"

### ISSUE THREAD

**Issue by scosma** [prometheus-community/json_exporter#88]
Hello,
I just deployed recently json_exporter.
We have some endpoints where we expose some metrics in json format, but apparently the json_exporter is complaining about it.

Here is the log from the exporter:
`level=error ts=2021-03-02T15:08:36.542Z caller=collector.go:63 msg="Failed to convert extracted value to float64" path="{ .results }" value="{\"cron-is-running\":{\"description\":\"\",\"duration\":0.0002779960632324219,\"name\":\"Cron is running\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"f-c-m-notifications\":{\"description\":\"There are successful sent notifications\",\"duration\":0.0017249584197998047,\"name\":\"FCM Notifications\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"guestbook-retention-was-processed\":{\"description\":\"\",\"duration\":0.0002741813659667969,\"name\":\"Guestbook retention was processed\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"news-synchronize\":{\"description\":\"\",\"duration\":0.0002570152282714844,\"name\":\"news-synchronize\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"queue-fail-jobs\":{\"description\":\"Checks if queue contains failed jobs\",\"duration\":0.0013568401336669922,\"name\":\"Queue fail jobs\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"queue-works\":{\"description\":\"Checks if queue job is processed last 3 minutes\",\"duration\":0.00030303001403808594,\"name\":\"Queue works\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"reports-retention-was-processed\":{\"description\":\"\",\"duration\":0.0002760887145996094,\"name\":\"Reports retention was processed\",\"reason\":\"\",\"status\":\"success\",\"tags\":[\"basic\"]},\"reservations-retention-was-processed\":{\"description\":\"\",\"duration\":0.0002760887145996094,\"name\":\"Reservations retention was processed\`

Our json looks like this:

`{"results":{"throttling":{"name":"Throttling","description":"Throttling limit","status":"success","reason":"","duration":0.0076601505279541016,"tags":["basic"]},"queue-fail-jobs":{"name":"Queue fail jobs","description":"Checks if queue contains failed jobs","status":"failed","reason":"There are 99 failed jobs. Please fix or forget them.","duration":0.016350984573364258,"tags":["basic"]},"f-c-m-notifications":{"name":"FCM Notifications","description":"There are successful sent notifications","status":"success","reason":"","duration":0.0019271373748779297,"tags":["basic"]},"scheduled-notifications":{"name":"Scheduled Notifications","description":"There are no waiting notifications","status":"success","reason":"","duration":0.0015687942504882812,"tags":["basic"]},"signage-a-p-i":{"name":"Signage API","description":"Signage API responses with code 2xx","status":"success","reason":"","duration":0.4234778881072998,"tags":["basic"]},"synchronizing-events":{"name":"Synchronizing events","description":"","status":"success","reason":"","duration":0.0007641315460205078,"tags":["basic"]},"synchronizing-transports":{"name":"Synchronizing transports","description":"","status":"success","reason":"","duration":0.0005240440368652344,"tags":["basic"]},"cron-is-running":{"name":"Cron is running","description":"","status":"success","reason":"","duration":0.0005209445953369141,"tags":["basic"]},"news-synchronize":{"name":"news-synchronize","description":"","status":"success","reason":"","duration":0.0004658699035644531,"tags":["basic"]},"guestbook-retention-was-processed":{"name":"Guestbook retention was processed","description":"","status":"success","reason":"","duration":0.0004639625549316406,"tags":["basic"]},"reservations-retention-was-processed":{"name":"Reservations retention was processed","description":"","status":"success","reason":"","duration":0.00037384033203125,"tags":["basic"]},"reports-retention-was-processed":{"name":"Reports retention was processed","description":"","status":"success","reason":"","duration":0.0002779960632324219,"tags":["basic"]},"queue-works":{"name":"Queue works","description":"Checks if queue job is processed last 3 minutes","status":"success","reason":"","duration":0.0003039836883544922,"tags":["basic"]}},"stats":{"time":0.4546797275543213,"time_per_test":0.034975363658024713,"number_of_tests":13},"succes`

the config of the exporter is pretty simple:
`---
metrics:
- name: lifecheck
  path: "{ .* }"
  help: scrapes the entire json
  labels:
    environment: lifecheck-inside-server`

What can I do to avoid this error?
Thanks

**Comment by snook9** [comment]
Same issue for me. Have you find a solution?

**Comment by scosma** [comment]
Yeah, we rewrote our Json file. Apparently this was the issue in our case.

**Comment by snook9** [comment]
OK. Thanks. 
Difficult for me because the json comes from an external service... :(

**Comment by scosma** [comment]
That sucks.

**Comment by snook9** [comment]
Solved for me, I had a dot '.' as extra.

**Comment by scosma** [comment]
Glad you solved it
